src/python/nimbusml/internal/utils/data_stream.py

```diff
@@ -460,6 +460,13 @@ def head(self, n=5, skip=0):
         (out_model, out_data, out_metrics, _) = graph.run(verbose=True, X=self)
         return out_data
 
+    @property
+    def schema(self):
```

Comment:
Get the unmodified schema here instead since this will currently not work correctly for vectorized columns.